horace_core/sqw/@DnDBase/copy.m

```diff
@@ -0,0 +1,7 @@
+function wout = copy(w)
+% Copy this DnD object or an array of DnD objects
+%
+%   >> wout = copy(w);
+
+wout = w;
+end
```

Comment:
SQWDnDBase is intentionally not a handle class (yet). If it needs to become one later it will be straightforward to update the code and copy function.
There is explicit handling in the SQW `copy()` method to manage the handle-class PixelData attribute. Additions similar to that will need to be added here if the support classes are handle-classes.